tsconfig.test.json

```diff
@@ -4,6 +4,6 @@
     "outDir": "dist-spec"
   },
   "include": [
-    "./spec/**/*"
+    "./spec/**/*", "./src/**/*.ts", "./data/**/*"
```

Comment:
I don't think the data directory is needed - unless JSON is imported into TypeScript (which is valid and allows the JSON to be typed), there's really no reason to include it as "source."